about.html

```diff
@@ -7,57 +7,92 @@
   <title>Sprint Challenge - About</title>
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+  <link rel="stylesheet" href="css/about.css">
 
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
+    <div class="container">
+
+      <header>
+        <div class="header">
+            
```

Comment:
For the header, you used a lot of code to create it. you can just use one header tag and nav tag and the results should be the same. Section tags are usually not used inside of header tags as well. Otherwise great creativity for the other links on the page.